src/pages/pt-BR/guides/imports.md

```diff
@@ -0,0 +1,151 @@
+
+## JavaScript
+
+```js
+import { getUser } from './user.js';
+```
+
+JavaScript pode ser importado usando a sintaxe normal de ESM `import` e `export`. Isto funciona como o esperado, baseado no comportamento padrão do Node.js e do Browser.
+
+## TypeScript
+
+```js
+import { getUser } from './user.ts';
+import type { UserType } from './user.ts';
+```
+
+Astro inclui suporte padrão para [TypeScript](https://www.typescriptlang.org/). Você pode importar arquivos `.ts` e `.tsx` diretamente em seu projeto Astro e até escrever código TypeScript dentro de seu [componente Astro](/pt-BR/core-concepts/astro-components/#script-do-componente).
+
+**Astro não realiza checagem de tipo**. A checagem de tipo deve ser feita fora do Astro, em sua IDE ou em scripts separados. A [Extensão Astro no VSCode](/pt-BR/editor-setup/) provê automaticamente dicas e erros de TypeScript em seus arquivos abertos.
+
+📚 Leia mais sobre o [suporte de TypeScript no Astro](/pt-BR/guides/typescript/).
+
+## JSX / TSX
+
+```js
+import { MeuComponente } from './MeuComponente.jsx';
+```
+
+Astro inclui suporte padrão para arquivos JSX (`*.jsx` and `*.tsx`) em seu projeto. A sintaxe JSX é automaticamente transpilada para JavaScript.
+
+Apesar do Astro entender a sintaxe JSX por padrão, você deverá incluir a integração de framework UI adequada para renderizar componentes React, Preact e Solid. Confira o nosso guia [Usando Integrações](/pt-BR/guides/integrations-guide) para saber mais.
+
+**Nota: Astro não da suporte à JSX em arquivos `.js`/`.ts`**. JSX será manipulado apenas dentro de arquivo que terminam com as extensões `.jsx` ou `.tsx`.
+
+## Pacotes NPM
+
+```js
+// Importa os pacotes npm React e React-DOM
+import React from 'react';
+import ReactDOM from 'react-dom';
+```
+
+Astro permite que você importe pacotes NPM diretamente no navegador. Mesmo se um pacote foi publicado usando um formato legado, Astro irá convertê-lo para ESM antes de serví-lo ao navegador.
+
+## JSON
+
+```js
+// Carrega o objeto JSON pelo "export" padrão.
+import json from './data.json';
+```
+
+Astro da o suporte para a importação de arquivos JSON diretamente em sua aplicação. Arquivos importados retornam o objeto JSON completo no `import` padrão.
+
+## CSS
+
+```js
+// Carrega e injeta o arquivo 'style.css' na página
+import './style.css';
+```
+
+Astro da o suporte para a importação de arquivos CSS diretamente em sua aplicação. Estilos importados não proveêm nenhum "export", mas importar um arquivo CSS automaticamente adiciona este estilo à página. Isto funciona para todos os arquivos CSS por padrão e pode suportar também os pré-processadores como Sass e Less via plugins.
+
+Se você prefere não escrever CSS, Astro também suporta todas as bibliotecas populares de CSS-in-JS (exemplo: styles-components) para estilização.
+
+## CSS Modules
```

Comment:
```suggestion
## Módulos CSS
```

Reply:
I don't know if we should translate this, I think this it's a common keyword like React, Vue and etc.

Reply:
My reasoning for translating CSS Modules, is that, AFAIK, CSS Modules are like a spec for different build and processing tools to implement, although not an official one. For example, `Selectors Level 4` are translated to `Seletores Nível 4` on a lot of websites I found. I don't really have strong opinions on this, feel free to decide.

Reply:
That's alright, I'll keep the translation! If we chang our mind, we can update this later.